appveyor.yml

```diff
@@ -7,62 +7,47 @@ branches:
 skip_tags: true
 clone_depth: 10
 
-init:
-  - mklink /d C:\git "C:\Program Files\Git"
-  - if %ruby_version%==_trunk (
-      appveyor DownloadFile https://ci.appveyor.com/api/projects/MSP-Greg/ruby-loco/artifacts/ruby_trunk.7z -FileName C:\ruby_trunk.7z &
-      7z x C:\ruby_trunk.7z -oC:\ruby_trunk )
-
 environment:
-  PATH: C:/ruby%ruby_version%/bin;C:/Program Files/7-Zip;C:/Program Files/AppVeyor/BuildAgent;C:/git/cmd;C:/Program Files (x86)/GNU/GnuPG/pub;C:/Windows/system32;C:\Windows;
-  WINDIR: C:\Windows
   matrix:
-  - ruby_version: 193
```

Comment:
why is this removed?

Reply:
@MSP-Greg could you try adding 1.9.3 back and seeing how it goes, since we do still support it? if you run into issues i'im happy to help you debug it sometime next week.